src/app/components/Search.js

```diff
@@ -62,6 +68,31 @@ const StyledSearchInput = styled.input`
   padding-${props => (props.isRtl ? 'right' : 'left')}: ${units(6)};
 `;
 
+const StyledPopper = styled(Popper)`
+  width: 100%;
+  max-width: ${columnWidthMedium};
+  padding: 0 ${units(1)};
+
+  table {
+    width: 100%;
+    display: block;
+  }
+
+  td {
+    padding: ${units(1)};
+  }
+
+  a {
+    font: ${caption};
+    padding-left: ${units(1)};
+  }
+
+  button {
+    color: ${black54};
+    float: right;
```

Comment:
Should it be aligned on the right for Arabic too? If not, please use different values based on `props.isRtl` like above